cloudberry/neo/app/actor/RequestRouter.scala

```diff
@@ -54,11 +55,11 @@ class RequestRouter (berryClientProp: Props, config: Config, requestHeader: Requ
   }
 
   private def handleNonStreamingBody(requestBody: JsValue, transform: IPostTransform): Unit = {
-    nonStreamingBerryClient ! Request(requestBody, transform)
+    nonStreamingBerryClient ! (requestBody, transform)
```

Comment:
Since `BerryClient` is now separating slicing and non-slicing queries, this `RequestRouter` class will be redundant right?

Reply:
Yes, it will be removed in the next PR.

Reply:
OK